{"data": "Traceback (most recent call last):\n  File \"/Users/akrah/Documents/dev/SemEval22/clozetask/utils/zlog.py\", line 39, in log_context\n    yield self\n  File \"/Users/akrah/Documents/dev/SemEval22/clozetask/management/runscript.py\", line 135, in run_loop\n    jiant_task_container = container_setup.create_jiant_task_container_from_json(\n  File \"/Users/akrah/Documents/dev/SemEval22/clozetask/management/components/container_setup.py\", line 256, in create_jiant_task_container_from_json\n    return create_jiant_task_container_from_dict(\n  File \"/Users/akrah/Documents/dev/SemEval22/clozetask/management/components/container_setup.py\", line 240, in create_jiant_task_container_from_dict\n    return create_jiant_task_container(\n  File \"/Users/akrah/Documents/dev/SemEval22/clozetask/management/components/container_setup.py\", line 202, in create_jiant_task_container\n    task_dict = create_task_dict(task_config_dict=task_config_path_dict, verbose=verbose)\n  File \"/Users/akrah/Documents/dev/SemEval22/clozetask/management/components/container_setup.py\", line 79, in create_task_dict\n    task = create_task_from_config_path(config_path=task_config_path, verbose=False)\n  File \"/Users/akrah/Documents/dev/SemEval22/clozetask/retrieval.py\", line 58, in create_task_from_config_path\n    return create_task_from_config(\n  File \"/Users/akrah/Documents/dev/SemEval22/clozetask/retrieval.py\", line 32, in create_task_from_config\n    task_class = get_task_class(config[\"task\"])\nTypeError: string indices must be integers\n", "TIMESTAMP": 1638132287.625021}
